internal/bgp/frr/parse.go

```diff
@@ -39,6 +39,7 @@ type FRRNeighbor struct {
 	BgpState          string       `json:"bgpState"`
 	PortForeign       int          `json:"portForeign"`
 	MsgStats          MessageStats `json:"messageStats"`
+	VRFName           string       `json:"vrfName"`
```

Comment:
sorry if it's silly, what does this do?

Reply:
it's a leftover, from when I thought I would've needed to introspect the frr containers to look for the vrf too, but from their point of view they will always consume the default vrf